platform/ios/src/MGLAnnotationView.h

```diff
@@ -13,7 +13,7 @@ NS_ASSUME_NONNULL_BEGIN
  @param reuseIdentifier The string that identifies that this annotation view is reusable.
  @return The initialized annotation view object or `nil` if there was a problem initializing the object.
```

Comment:
I agree and thanks for helping me understand this a bit better. I'm going to remove the confusing language in the comment. Seems like MK and older MB code needs some additional comment cleaning (i.e. https://github.com/mapbox/mapbox-gl-native/blob/master/platform/ios/src/MGLAnnotationImage.h#L17)